### PR TITLE
Pop extra '(' from active filters [fixes #150876974]

### PR DESCRIPTION
**Pull request by timo-piirmann**

### CODE REVIEW

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy [hunk 1]
```diff
@@ -142,19 +142,21 @@ class OccurrenceTagLib {
...
+                        .replaceAll(':', ': ')
                         .replaceAll(
                             'occurrence_year',
                             alatag.message(code: 'facet.occurrence_year', default:'occurrence_year')
                         )
+                        .replaceAll(/(\d{4})\-.*?Z/)
 
-                    mkp.yieldUnescaped(fqLabel.replaceAll(/(\d{4})\-.*?Z/) { all, year ->
+                    mkp.yieldUnescaped(fqLabel { all, year ->
                         def year10 = year?.toInteger() + 10
```
Comment (alarku07) [inline grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: isn't this going into exception if year can't be integerized?

Comment (timo-piirmann) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Nick has probably handled that somewhere

Comment (filippivanov) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: ALA policy on input validation is at best "crash with 500"

Comment (timo-piirmann) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: It's this way for logging purposes

**Pull request merged**